cellprofiler/gui/figure.py

```diff
@@ -1096,7 +1096,11 @@ def gridshow(self, x, y, image, cmap='gray'):
             if position % 3 != 0:
                 ax.set_yticklabels([])
 
-            ax.imshow(image[position * (z - 1) / 8], cmap=cmap, norm=matplotlib.colors.LogNorm(vmin=vmin, vmax=vmax))
+            ax.imshow(
+                image[position * (z - 1) / 8],
+                cmap=cmap,
+                norm=matplotlib.colors.SymLogNorm(linthresh=0.03, linscale=0.03, vmin=vmin, vmax=vmax)
```

Comment:
The data in range [0.0, 0.03] is normalized on a linear scale instead of a logarithmic scale.
<img width="857" alt="screen shot 2016-12-14 at 9 49 47 am" src="https://cloud.githubusercontent.com/assets/4721755/21186488/d6aa1fe6-c1e2-11e6-805d-8cd05ad4476a.png">